index.js

```diff
@@ -71,9 +78,18 @@ function getName(/* code here */) {
  *         and returns a string like `Hello, my name is {name}`.
  *         where `{name}` is the name passed into `makeSmartPerson`.
 */
-function makeSmartPerson(/* code here */) {
-  /* code here */
-}
+function makeSmartPerson(name) {
+  const smartPerson= {
+    name: name,
+    sum: function(num1, num2){
+      return num1 + num2;
+    },
+    speak: function(){
+      return "Hello, my name is " + name;
+    }
+  } //end object
+  return smartPerson;
+}//end function
 
```

Comment:
This would be a great place for arrow functions on lines 84 to 89.  Why? Because arrow functions are great when the function is anonymous (meaning it is not named) and it only has one statement inside it.